packages/commerce-sdk-react/package.json

```diff
@@ -57,7 +57,7 @@
     "@types/react-helmet": "^6.1.6",
     "cross-env": "^5.2.1",
     "internal-lib-build": "3.0.0-dev",
-    "jsonwebtoken": "^8.5.1",
+    "jsonwebtoken": "^9.0.0",
```

Comment:
Should we do the same for the @type: https://www.npmjs.com/package/@types/jsonwebtoken